docs/serving/samples/hello-world/helloworld-csharp/Dockerfile

```diff
@@ -1,18 +1,19 @@
 # Use Microsoft's official .NET image.
 # https://hub.docker.com/r/microsoft/dotnet
-FROM microsoft/dotnet:2.2-sdk
-
-# Install production dependencies.
-# Copy csproj and restore as distinct layers.
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS build
 WORKDIR /app
+
+# copy csproj and restore as distinct layers
 COPY *.csproj .
```

Comment:
The `COPY *.ext .` pattern will eventually bite non-experts (this works because there's only 1 `csproj`).
→ I'd suggest using `COPY *.csproj ./` instead.

Reply:
Nice catch. I never knew this.

> If multiple <src> resources are specified, either directly or due to the use of a wildcard, then <dest> must be a directory, and it must end with a slash /